tests/unit/test_exceptions.py

```diff
@@ -14,35 +14,37 @@
 
 import http.client
 import json
+from google.rpc import status_pb2
+from google.rpc.status_pb2 import Status
 
 import grpc
 import mock
 import requests
-
+from google.rpc import error_details_pb2
```

Comment:
This import needs to be conditional:
```suggestion
try:
    from google.rpc import error_details_pb2
except ImportError:
    details_pb2 = None
```